client/lib/plans/features-list.tsx

```diff
@@ -2168,6 +2194,139 @@ export const FEATURES_LIST: FeatureList = {
 			i18n.translate( 'Keep an administrative eye on activity across your site.' ),
 	},
 	/* END: 2023 Pricing Grid Features */
+
+	/* START: Woo Express Features */
+	[ FEATURE_WOOCOMMERCE_STORE ]: {
+		getSlug: () => FEATURE_WOOCOMMERCE_STORE,
+		getTitle: () => 'WooCommerce store',
```

Comment:
Do all these `getTitle` values need to be wrapped in translate functions? I see some are and some are not.

Reply:
They definitely should be wrapped in translate functions. Thanks!